plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerLaunchConfigurationDelegate.java

```diff
@@ -117,10 +117,17 @@ private static void validateCloudSdk() throws CoreException  {
   @Override
   public ILaunch getLaunch(ILaunchConfiguration configuration, String mode) throws CoreException {
     IServer thisServer = ServerUtil.getServer(configuration);
-    DefaultRunConfiguration thisConfig = generateServerRunConfiguration(configuration, thisServer);
+    return getLaunchInternal(configuration, mode, thisServer, getLaunchManager());
```

Comment:
Good idea! I was doing this mechanically and syntactically, not trying to understand what this part does.